src/linker/Linker/Driver.cs

```diff
@@ -1059,6 +1084,8 @@ static void Usage ()
 			Console.WriteLine ("  -out PATH                     Specify the output directory. Defaults to 'output'");
 			Console.WriteLine ("  --about                       About the {0}", _linker);
 			Console.WriteLine ("  --verbose                     Log messages indicating progress and warnings");
+			Console.WriteLine ("  --warn VERSION                Only print out warnings with version <= VERSION. Defaults to '9999'");
```

Comment:
This description does not make much sense. We should be explicit about what is supported and that's right now the only version 5